op-challenger/fault/monitor_test.go

```diff
@@ -86,15 +87,15 @@ func setupMonitorTest(t *testing.T, allowedGame common.Address) (*gameMonitor, *
 	fetchBlockNum := func(ctx context.Context) (uint64, error) {
 		return 1234, nil
 	}
-	monitor := newGameMonitor(logger, fetchBlockNum, allowedGame, source, games.CreateGame)
+	monitor := newGameMonitor(logger, time.Duration(int64(0)), fetchBlockNum, allowedGame, source, games.CreateGame)
```

Comment:
Golang automatically casts untyped integer constants to the right type.
```suggestion
	monitor := newGameMonitor(logger, time.Duration(0), fetchBlockNum, allowedGame, source, games.CreateGame)
```